webapp/src/utils/filterData.ts

```diff
@@ -19,4 +19,9 @@ export const getProjectList = (data: ProjectDetails[]) => {
     return projectList;
 };
 
-export const getCurrentChannelSubscriptions = (data: SubscriptionDetails[], channelID: string): SubscriptionDetails[] => data.filter(((subscription) => subscription.channelID === channelID));
+export const getCurrentChannelSubscriptions = (data: SubscriptionDetails[], channelID: string) => (data || []).filter(((subscription) => subscription.channelID === channelID));
+
+export const getCurrentChannelName = (data: ChannelList[], channelID: string) => {
+    const currentChannel = (data || []).filter(((channel) => channel.id === channelID));
+    return currentChannel[0]?.display_name;
+};
```

Comment:
```suggestion
export const getCurrentChannelName = (data: ChannelList[], channelID: string) => data.find((channel) => channel.id === channelID)?.display_name;
```

It was fixed by @ayusht2810, I think it is not in sync with that branch.

Reply:
this is a change of @ayusht2810 can you look into this and try replacing this with the one which you have fixed or is more recent in any branch